### PR TITLE
feat(Boost): add complete action to Boost plugin

### PR DESCRIPTION
**Pull request by jonathandiep**

### CODE REVIEW

File: packages/boost/src/Boost.ts [hunk 1]
```diff
@@ -2,14 +2,85 @@ import {
...
+import axios from 'axios'
 import {
   BOOST_PASS_CONTRACT,
   BOOST_PASS_ABI,
   DATA_ABI_PARAMS,
 } from './constants'
+import { CompletedBoostResponse, CompletedBoostResponseSchema } from './types'
+
+// const API_BASE_URL = 'https://api.boost.xyz'
+const API_BASE_URL = 'http://localhost:5000'
```
Comment (jonathandiep) [inline packages/boost/src/Boost.ts:24]: Use prod api

File: packages/boost/rollup.config.mjs [hunk 2]
```diff
@@ -40,7 +40,6 @@ export default {
     },
   ],
   plugins: [
-    polyfillNode(),
```
Comment (Quazia) [inline packages/boost/rollup.config.mjs:43]: We're trying to move over to Vite for our build now (sorry for the whiplash) but why remove this for this PR?

File: packages/utils/src/types/actions.ts [hunk 3]
```diff
@@ -298,29 +298,29 @@ export type CreateActionDetail = z.infer<typeof CreateActionDetailSchema>
...
-  })
-  .refine(
-    (data) =>
-      data.chainId != null || data.boostId != null || data.actionType != null,
-    {
-      message:
-        'At least one of chainId, boostId, or actionType must be provided',
-    },
-  )
+export const CompleteActionDetailSchema = z.object({
```
Comment (Quazia) [inline packages/utils/src/types/actions.ts:307]: We can rely on this export once the utils are cleaned up vs copying it over the backend. No action needed mostly a reminder for me to go back and clean this up in the RH repo once that change is live.

**Review by Quazia — APPROVED** [review]
Looks good - let me know if you want help testing this flow with the corresponding backend changes.

**Review by jimobrien — APPROVED** [review]
🔥

File: packages/boost/rollup.config.mjs (hunk 2 above)
Comment (jonathandiep) [reply packages/boost/rollup.config.mjs:43]: There was some weird build config (I think it was after importing axios into the boost plugin)

At some point, I took at look at the Neynar plugin and it didn't have the `polyfillNode()`. So I removed it here too and it worked

**Pull request merged**